### DISCUSSION TITLE
iOS support

### DISCUSSION THREAD

**Discussion by Clafou** [oven-sh/bun#9436] [Ideas]
It would be great to be able to embed Bun in an iOS app.

JavaScriptCore is available in iOS, but it is very limited without IO and modules such as `net`, `fs` etc.

**Comment by theolampert** [comment]
@Clafou I attempted to expose some more APIs to JSC via briding to Swift and Apple frameworks here https://github.com/theolampert/ECMASwift with varying results, the implementation isn't complete. Ideally, either Bun supporting iOS or a project that makes it simple to expose some more of WebCore's APIs to JSC as Bun does would be really useful.
